.github/workflows/judoscale-que-test.yml

```diff
@@ -14,18 +14,29 @@ jobs:
       matrix:
         gemfile:
           - Gemfile
+          - Gemfile-activerecord-7-0
           - Gemfile-activerecord-6-1
-          - Gemfile-que-2
+          - Gemfile-que-1
```

Comment:
Swapped the Gemfile to tests Que v1, since v2 is tested by the main Gemfile (and the alternative AR versions)